src/renderer/components/hotbar/hotbar-cell.tsx

```diff
@@ -0,0 +1,36 @@
+import "./hotbar-menu.scss";
+import "./hotbar.commands";
+
+import React, { HTMLAttributes, ReactNode, useState } from "react";
+
+import { cssNames } from "../../utils";
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  children?: ReactNode;
+  index: number;
+  innerRef?: React.LegacyRef<HTMLDivElement>;
+}
+
+export function HotbarCell({ innerRef, children, className, ...rest }: Props) {
+  const [animating, setAnimating] = useState(false);
+  const onAnimationEnd = () => { setAnimating(false); };
+  const onClick = () => {
+    if (className.includes("isDraggingOver")) {
+      return;
+    }
+
+    setAnimating(true);
```

Comment:
I think the following would have the same effect.

```suggestion
    setAnimating(!className.includes("isDraggingOver"));
```